src/main.ts

```diff
@@ -131,18 +131,21 @@ function loadTscConfig(args: string[]):
 export function toClosureJS(
     options: ts.CompilerOptions, fileNames: string[], settings: Settings,
     writeFile?: ts.WriteFileCallback): tsickle.EmitResult {
-  const compilerHost = ts.createCompilerHost(options);
-  const program = ts.createProgram(fileNames, options, compilerHost);
   // Use absolute paths to determine what files to process since files may be imported using
   // relative or absolute paths
-  const filesToProcess = new Set(fileNames.map(i => path.resolve(i)));
+  const absoluteFileNames = fileNames.map(i => path.resolve(i));
+
+  const compilerHost = ts.createCompilerHost(options);
+  const program = ts.createProgram(fileNames, options, compilerHost);
+  const filesToProcess = new Set(absoluteFileNames);
   const transformerHost: tsickle.TsickleHost = {
     shouldSkipTsickleProcessing: (fileName: string) => {
       return !filesToProcess.has(path.resolve(fileName));
     },
     shouldIgnoreWarningsForPath: (fileName: string) => false,
     pathToModuleName: cliSupport.pathToModuleName,
```

Comment:
what about binding the parameter here, so we don't have to pipe rootModulePath through all the system?

```
const rootModulePath = getCommon...;
// ...
  pathToModuleName: (context: string, importPath: string) => cliSupport.pathToModuleName(context, importPath, rootModulePath),
```

That saves us a lot of piping things around, but achieves the same thing I think?